test/dom/propertyShadowing.ts

```diff
@@ -1,19 +1,25 @@
 import {assert} from "chai";
 import {dom, IonTypes} from "../../src/Ion";
+import {Err} from "typedoc/dist/lib/utils/result";
```

Comment:
Is this import necessary? It looks like it's using a type from the `typedoc` package.